spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -293,13 +300,14 @@ def get_pending_release_app_store_version(client: nil, platform: nil, includes:
         return get_app_store_versions(client: client, filter: filter, includes: includes).first
       end
 
-      def get_app_store_versions(client: nil, filter: {}, includes: Spaceship::ConnectAPI::AppStoreVersion::ESSENTIAL_INCLUDES, limit: nil, sort: nil)
+      def get_app_store_versions(client: nil, filter: {}, includes: Spaceship::ConnectAPI::AppStoreVersion::ESSENTIAL_INCLUDES, limit: nil)
```

Comment:
👍  A quick search in the codebase seem to confirm that this method was never called with a value for the `sort:` parameter before anyway, so removing it shouldn't be a breaking change to remove that parameter 👍

Reply:
Could that break external users though? `spaceship` is also a module.

We could do one thing.
1. Keep the parameter for a while
2. log with deprecated if the value isn't `nil`
3. delete the parameter in X months.

I haven't seen a proper strategy for managing deprecation / API stability yet.